h2o-automl/src/main/java/ai/h2o/automl/events/EventLog.java

```diff
@@ -19,16 +18,16 @@
  */
 public class EventLog extends Keyed<EventLog> {
 
-  public final Key<AutoML> _automl_id;
+  public final Key _runner_id;
```

Comment:
renamed this as it is not not used only for automl, for also for selection, the same applies with Leaderboard.
This is mainly because EventLog is a property of leaderboard, and I need a temporary leaderboard to apply the selection logic.